Resources/ServerInfo/Guidebook/Slang.xml

```diff
@@ -0,0 +1,35 @@
+<Document>
+  # Slang Terms
+  Space Station 14 players tend to use a lot of slang in their language to cut down on time spent typing.
+
+  ## Common IC (In Character) terms
+  - [color=#a4885c]Atmos[/color]: Shorthand for the atmospherics department.
+  - [color=#a4885c]AOS/KOS[/color]: Shorthand for Attack On Sight and Kill On Sight respectively. Both terms are commonly used by security.
```

Comment:
pretty sure AOS stands for arrest on sight, and not attack on sight

Reply:
that is at least how i use it, and have seen people use AOS

Reply:
Arrest on Sight
Kill on Sight